edit_form.php

```diff
@@ -0,0 +1,36 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Block edit form class for the block_iagora plugin.
+ *
+ * @package   block_iagora
+ * @copyright 2024, Datum Academy <dev@datum.academy>
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+class block_iagora_edit_form extends block_edit_form {
+    protected function specific_definition($mform) {
+        // Section header title according to language file.
+        $mform->addElement('header', 'config_header', get_string('blocksettings', 'block'));
+
+        // URL de l'iframe
+        $mform->addElement('text', 'config_iframeurl', get_string('iframeurl', 'block_iagora'));
```

Comment:
```suggestion
        $mform->addElement('text', 'config_iframeurl', get_string('iframe_url', 'block_iagora'));
```
The string should match the one defined in `lang/en/block_iagora.php`.

What about renaming it to `copilot_iframe_url`?

Reply:
UPD: With Moodle naming convention, it would be `copilotiframeurl`, which might be less readable; let's keep `iframeurl`.

Reply:
I will renomer respecting the Moodle naming convention.